src/Resources/skeleton/bundle/bundle.mustache

```diff
@@ -14,11 +14,4 @@ use Symfony\Component\HttpKernel\Bundle\Bundle;
  */
 class {{ application }}{{ bundle }} extends Bundle
 {
-    /**
```

Comment:
This is not a technical BC break, but a functional if you are using symfony < 4

Reply:
What do you mean by that?

Reply:
It already deprecated in 3.* symfony
And didn't had any effect on 2.* symfony

Reply:
> What do you mean by that?

It's not a BC break like when we drop a required argument in a class method. But it is a feature BC break, because now bundle inheritance is removed by default.

Reply:
> because now bundle inheritance is removed by default.

Yes but what does this imply? It will not break existing code as I established before, but will it break new code?

Reply:
as in the symfony blog post i menthioned in the issue, it never worked before for anything sonata does with the Entity/Model classes

Reply:
> Yes but what does this imply? It will not break existing code as I established before, but will it break new code?

It will break code if you use a LTS release of symfony and need bundle inheritance

Reply:
While having already generated the app you mean?

Reply:
No, when calling it in an old symfony instance. As said, we remove a feature with this PR.

Reply:
@Core23 https://symfony.com/blog/new-in-symfony-3-4-deprecated-bundle-inheritance
Non of the stuff sonata did with bundle inheritence works with old and new symfony, and if the user use old symfony and still need inheritance he might define it himself

Reply:
@Hanmac Does this mean I can just remove the `getParent()` function in my existing Symfony 3.3 app when upgrading to 3.4 and everything will work fine?

I have two SonataEasyExtendsBundle generated bundles (`Application\Sonata\ClassificationBundle` and `Application\Sonata\MediaBundle`, first created way back in time, may be like in Symfony 2.3).

Reply:
@enekochan for your 3.3 and the updated 3.4 version getParent would have no effect
(other than showing warning)
so you chould just remove the functions

Reply:
And what about If I am using Symfony 2.8 and generate this bundle without inheritance?

Reply:
I'm okay with this as long as you leave a short feature BC break note.

Reply:
@core23 a break note inside the bundle.mustache  or where should i put it and how should it look like?

Reply:
Inside the https://github.com/sonata-project/SonataEasyExtendsBundle/blob/2.x/UPGRADE-2.x.md file please